sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -204,14 +204,23 @@
  *
  * <h3>Partition Assignment and Checkpointing</h3>
  * The Kafka partitions are evenly distributed among splits (workers).
- * Checkpointing is fully supported and each split can resume from previous checkpoint. See
- * {@link UnboundedKafkaSource#split(int, PipelineOptions)} for more details on
+ *
+ * <p>Checkpointing is fully supported and each split can resume from previous checkpoint,
+ * if it's supported and enabled in runner configuration.
```

Comment:
This could be rephrased [Optional]:
"Checkpointing is fully supported and each split can resume from previous checkpoint (_to the extent supported a runner_)"...